### PR TITLE
Handles ctrl+S in monaco editor and displays a snackbar.

### PR DESCRIPTION
**Pull request by jatinAroraGit**
Hey there,
Now a snackbar with message 'Saved' is displayed instead of Save Page dialog box when Ctrl+S is pressed.
The snackbar implemented is MatSnackBar from angular/material package

### CODE REVIEW

File: libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts [hunk 1]
```diff
@@ -71,10 +73,26 @@ export class EditorFromModelComponent implements AfterViewInit, OnDestroy {
...
 
+    this.editor.addAction({
+      id: 'saveAction',
+      label: 'Save Shortcut Press',
+      keybindings: [
+        this.monacoConfigService.monaco.KeyMod.CtrlCmd | this.monacoConfigService.monaco.KeyCode.F10,
+        // chord
+        this.monacoConfigService.monaco.KeyMod.chord(this.monacoConfigService.monaco.KeyMod.CtrlCmd | this.monacoConfigService.monaco.KeyCode.KEY_S)
+      ],
+      run: function (ed) {
```
Comment (kirjs) [inline libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts]: can you pls use arrow function here and avoid `that`?

File: libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts [hunk 2]
```diff
@@ -71,10 +73,26 @@ export class EditorFromModelComponent implements AfterViewInit, OnDestroy {
...
     this.editor = this.editorInjector.editor = this.setUpEditor(
       this.el.nativeElement
     );
 
+    this.editor.addAction({
+      id: 'saveAction',
+      label: 'Save Shortcut Press',
+      keybindings: [
+        this.monacoConfigService.monaco.KeyMod.CtrlCmd | this.monacoConfigService.monaco.KeyCode.F10,
+        // chord
```
Comment (kirjs) [inline libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts]: Is there much value in this comment?

File: libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts [hunk 3]
```diff
@@ -71,10 +73,26 @@ export class EditorFromModelComponent implements AfterViewInit, OnDestroy {
...
+    const that = this;
     this.editor = this.editorInjector.editor = this.setUpEditor(
       this.el.nativeElement
     );
 
+    this.editor.addAction({
+      id: 'saveAction',
+      label: 'Save Shortcut Press',
+      keybindings: [
+        this.monacoConfigService.monaco.KeyMod.CtrlCmd | this.monacoConfigService.monaco.KeyCode.F10,
```
Comment (kirjs) [inline libs/code-demos/src/lib/multitab-editor/editor-from-model/editor-from-model.component.ts]: Curious what's f10 here for?

**Comment by jatinAroraGit** [comment]
Hey @kirjs , I addressed the issues in the new commit. Let me know if there are any more issues.

**Comment by NothingEverHappens** [comment]
🔥
The code looks good, please sync and format (npm run format:write) and we can submit

**Comment by jatinAroraGit** [comment]
@NothingEverHappens  Did another commit with everything Synced and addressed the requested changes.

**Comment by kirjs** [comment]
Amazing, thanks a lot @jatinAroraGit !

**Pull request merged**